imap_processing/codice/constants.py

```diff
@@ -12,8 +12,7 @@
 ESA = ElectroStatic Analyzer
 """
 
-# TODO: What to do in the case of a value of 255 in LOSSY_A and LOSSY_B
-#       compression? (Joey uses 0x100000000)
+from collections import OrderedDict
```

Comment:
Do you need an OrderedDict? Python dictionaries are now order by default.
https://docs.python.org/3/library/collections.html#ordereddict-objects

Reply:
Ah, good to know! In that case I believe I can use just a regular dictionary